services/src/messaging/src/main/java/org/openkilda/messaging/model/DiscoveryLink.java

```diff
@@ -0,0 +1,251 @@
+package org.openkilda.messaging.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+@JsonSerialize
```

Comment:
JsonSerialize annotation is redundant if you don't set specific serializer or other serialization aspects.